challenges/objects-arrays.js

```diff
@@ -59,11 +94,24 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+for(i=0; i<graduates.length;i++){
+	contactInfo.push(`${graduates[i].first_name} ${graduates[i].email} `);
+}
+
 console.log(contactInfo);
 
 
-/* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
+/* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result.*/
+
```

Comment:
I would like to challenge you (when you have time) to look up .includes and see how you could use it on this problem